web_sheet_full_width_selective/models/model.py

```diff
@@ -0,0 +1,59 @@
+# coding: utf-8
```

Comment:
and having 50 class files in a folder was doable with Java because you had the one click IDE cross file navigation when clicking on a method, but that's something we dont' have for Odoo Python dev...